### PR TITLE
Add subscription management option to embed

### PR DESCRIPTION
**Pull request by stianpr**
When creating an embed session one can now specify the option to `enable_subscription_management`.

### CODE REVIEW

**Review by fdanielsen — CHANGES_REQUESTED** [review]
The `README.rst` formatting is mangled. Column separators are injected in the middle of column contents.

**Comment by stianpr** [comment]
> The README.rst formatting is mangled. Column separators are injected in the middle of column contents.

Should be fixed now!

**Review by fdanielsen — CHANGES_REQUESTED** [review]
`README.rst` is not quite fixed yet. The HTTP path is at the beginning of the "Description" column, while it should be a part of the "HTTP request" column. Eg. `**POST** /accounts/` should be the "HTTP request" content, while "Create an account" should be the "Description".

**Comment by fdanielsen** [comment]
Regarding issues with generating correct readme.. Are you following the suggested procedure in the output from the `generate.sh` script in MailMojo?

> echo
> echo "After generating Python, first align the table in README.md and then run pandoc to generate README.rst:"
> echo "pandoc --from=markdown --to=rst --output=README.rst README.md"

**Comment by stianpr** [comment]
> Regarding issues with generating correct readme.. Are you following the suggested procedure in the output from the generate.sh script in MailMojo?

Yes of course I am. But I'm not sure what "aligning" means. So I have space aligned all the columns correctly. 

But If you can push the "View" button on the diff for the readme file it looks fine?
https://github.com/eliksir/mailmojo-python-sdk/blob/546c6a088b1526c9de969cef75cf7fd7cdef93ec/README.rst

**Comment by stianpr** [comment]
>

> README.rst is not quite fixed yet. The HTTP path is at the beginning of the "Description" column, while it should be a part of the "HTTP request" column. Eg. **POST** /accounts/ should be the "HTTP request" content, while "Create an account" should be the "Description".

Ahh i'm prolly missing a column then. I'll fix.

**Comment by fdanielsen** [comment]
You should try the ["rich diff" view](https://github.com/eliksir/mailmojo-python-sdk/pull/3/files?short_path=88b99bb#diff-88b99bb28683bd5b7e3a204826ead112) of that file then, and you'll see the erronous change quite easily :)

**Comment by stianpr** [comment]
> You should try the "rich diff" view of that file then, and you'll see the erronous change quite easily :)

Yes. Should be fixed now.

File: README.rst [hunk 1]
```diff
@@ -139,14 +139,14 @@ mailmojo\_auth
 -  **Authorization URL**: https://api.mailmojo.no/oauth/authorize/
 -  **Scopes**:
 
-  -  **account**: Manage your MailMojo account.
```
Comment (fdanielsen) [inline README.rst]: This "de-indent" of scopes list also does not seem correct. The list is a sub-list of the **Scopes** list item.

**Review by fdanielsen — CHANGES_REQUESTED** [review]
The changes in the list of scopes should also be reverted.

**Comment by stianpr** [comment]
> The changes in the list of scopes should also be reverted.

Fixed

**Review by fdanielsen — APPROVED** [review]

**Pull request merged**